src/lib/y2storage/dasd.rb

```diff
@@ -0,0 +1,50 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+require "y2storage/partitionable"
+
+module Y2Storage
+  # A DASD (direct-access storage device), typically used in mainframes
```

Comment:
so on what is based our knowledge? What we use as authoritative source? notes from IBM guys in fate? or Ihno only? or if we use kernel module for dasd, maybe that module have documentation?
What if someone want to touch this class, is it his task to "magically conjure up documentation"? we should help other, even poor documentation is better then none. E.g. https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.idad400/dasda.htm is at least something.